plutus-core/untyped-plutus-core/src/UntypedPlutusCore.hs

```diff
@@ -24,4 +26,10 @@ import PlutusCore.Name as Export
 applyProgram :: Program name uni fun () -> Program name uni fun () -> Program name uni fun ()
 applyProgram (Program _ _ t1) (Program _ _ t2) = Program () (PLC.defaultVersion ()) (Apply () t1 t2)
 
+-- | Turn a UPLC term to a UPLC program.
+mkProg :: Term name uni fun () -> Program name uni fun ()
+mkProg = Program () (PLC.defaultVersion ())
```

Comment:
I hate this whole `defaultVersion` business, we should remove it entirely some time.

Reply:
Maybe give it a nastier name. `mkDefaultProg` or something. Comment would be good. I'm not quite aggressive enough to suggest marking it as deprecated :D